docs/api.md

```diff
@@ -315,6 +318,30 @@ Shortcut for [page.mainFrame().click(selector[, options])](#frameclickselector-o
 #### page.close()
 - returns: <[Promise]>
 
+#### page.cookies(...urls)
+- `...urls` <...[string]>
+- returns: <[Promise]<[Array]<[Object]>>>
+  - `name` <[string]>
+  - `value` <[string]>
+  - `domain` <[string]>
+  - `path` <[string]>
+  - `expires` <[number]> Unix time in seconds.
+  - `httpOnly` <[boolean]>
+  - `secure` <[boolean]>
+  - `sameSite` <[string]> `"Strict"` or `"Lax"`.
+
+If no URLs are specified, this method returns cookies related to the current page.
+If URLs are specified, only cookies for those URLs are returned.
+
+#### page.deleteCookie(...cookies)
+- `...cookies` <...[Object]>
+  - `name` <[string]> **required**
+  - `url` <[string]>
```

Comment:
Either url or domain should be required for scoping. Also, this method will delete multiple cookies for each pattern, so might be worth calling method `deleteCookies` and parameters `patterns`